### DISCUSSION TITLE
LogIn into the German railway WEB.site not possible

### DISCUSSION THREAD

**Discussion by aage-2** [keepassxreboot/keepassxc#10305] [Q&A] [answered]
I have an account on the German railway system (https://www.bahn.de/). When I click on LogIn this changes to https://accounts.bahn.de/auth/realms/db/protocol/openid-connect/auth?redirect_uri=https%3A%2F%2Fwww.bahn.de%2F&client_id=kf_web&response_type=code&state=2miBMe7VzP&scope=openid%20vendo&response_mode=fragment&kc_locale=de&showRegisterWithBahnCard=true&code_challenge=-LriCH2C2egOzCz1_7ALjORY8H1v8ceqWiHvtlfFLx4&code_challenge_method=S256

I tried both Logins https://www.bahn.de/ and https://accounts.bahn.de/ none worked. If I use the long URL, KeePass paints it in red, so it is not acceptable. Also, I think, that one should always use short URLs. 
Where is my error, asks
Aage

**Comment by droidmonkey** [comment, marked as answer]
You need to go into the extension options then do the following:

![image](https://github.com/keepassxreboot/keepassxc/assets/2809491/638ab5ab-5bc1-469d-a7b0-495a8a6b7f56)

**Reply by aage-2** [reply]
Thanks droidmonkey, 
I am using Keepass only since ~10 days, and I have problems to follow your suggestions. 
1) My Keepass version was in German, and I found no way to go to the German version of the <extension options>. So I deleted the German version and downloaded the English one. But it starts automatically in German. Question: How do I change - in Germany - to the English version?
2) Your added picture shows the opened KEEPASS BROWSER. How do I open it? It doesn't show up in my APP list. 
3) I have successfully enterrd more than 10 URLs into KEEPASS - without any problems. But you see, I am a real beginner with KeePass.

Aage-2

WIN10 PRO-64

**Reply by droidmonkey** [reply]
Are you using the browser extension?

**Comment by aage-2** [comment]
Hello droidmonkey, 
I did as you suggested.  But still - something is wrong, see attachments. No login with Keepass, but a normal login with the **same** PW and user name is possible from Firefox.  What's wrong?
Asks,  Aage
![DB-login-black](https://github.com/keepassxreboot/keepassxc/assets/160011433/754e34f8-1318-4dcf-a76b-954f5e55d96b)
![DB-error](https://github.com/keepassxreboot/keepassxc/assets/160011433/1ed795bc-39db-4910-9e33-604ee044ca86)

**Reply by droidmonkey** [reply]
That error means you don't have any entries, that are available to the browser plugin, with that url. Re create your entry for this website, use the `https://accounts.bahn.de` url, try again.

**Comment by aage-2** [comment]
I recreated the url twice, with the same negative result, and the same earlier error message. Could this be a bug? The second time, I copied  url and password from the Keepass site and pasted into the accounts://bahn.de WEB site  --> negative.   As I said earlier, I can manually input username and password, and then the login is immediate. Really perplexing.

**Reply by droidmonkey** [reply]
It works perfectly fine for me, unfortunately

**Comment by aage-2** [comment]
Well, thanks anyway. I learned quite a bit. Do you have any ideas on what I could try? 
The next time I come about such a long URL, I will put into KeePass, just to see if the behaviour is similar.
